graql/language/match.feature

```diff
@@ -1238,7 +1238,7 @@ Feature: Graql Match Query
       | calories    | long     | 1761       |
       | grams       | double   | 9.6        |
       | gluten-free | boolean  | false      |
-      | use-by-date | datetime | 2020-06-16 |
+      | use-by-date | datetime | 2021-06-16 |
```

Comment:
let's revert it

Reply:
@flyingsilverfin @lolski addressed in https://github.com/graknlabs/behaviour/pull/136/commits/98f340f72238de28ce14dc2f629c82b1b1fc6242